data/data/agent/files/usr/local/share/assisted-service/assisted-service.env.template

```diff
@@ -20,3 +20,4 @@ STORAGE=filesystem
 INFRA_ENV_ID={{.InfraEnvID}}
 EC_PUBLIC_KEY_PEM={{.PublicKeyPEM}}
 AGENT_AUTH_TOKEN={{.Token}}
+AGENT_WORKFLOW_TYPE={{.AgentWorkflowType}}
```

Comment:
Shouldn't be required, see this [comment](https://github.com/openshift/assisted-service/pull/6605/files#r1689764220)